bin/node-template/flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, ... }: flake-utils.lib.eachDefaultSystem (system:
+    let
+      pkgs = import nixpkgs { inherit system; };
+    in
+    {
+      devShells.default = pkgs.mkShell {
+        packages = with pkgs; [
+          rustup
+          clang
+          protobuf
+          gnumake
```

Comment:
We don't need make?

Reply:
Here's a funny one. It's already in the `mkShell`. I used devenv.sh initially and frontier actually needs make to be built. When I was preparing the pull request, I cleaned up extra deps and removed devenv as well (as it seemed a simple mkShell would do the trick just fine). But mkShell pulls the basic builders implicitly, so make is still part of the derivation.

TL;DR: we don't need make here. We still need make to build frontier, but this derivation is universal and will contain make.